test/example_data.dart

```diff
@@ -117,12 +118,57 @@ ZulipStream stream({
     isWebPublic: isWebPublic ?? false,
     historyPublicToSubscribers: historyPublicToSubscribers ?? true,
     messageRetentionDays: messageRetentionDays,
-    streamPostPolicy: streamPostPolicy ?? 1,
-    canRemoveSubscribersGroupId: canRemoveSubscribersGroupId ?? 123,
+    streamPostPolicy: streamPostPolicy ?? StreamPostPolicy.any,
+    canRemoveSubscribersGroup: canRemoveSubscribersGroup ?? 123,
+    streamWeeklyTraffic: streamWeeklyTraffic,
   );
 }
 const _stream = stream;
 
+/// Construct an example subscription from a stream.
+///
+/// We only allow overrides of values specific to the [Subscription], all
+/// other properties are copied from the [ZulipStream] provided.
+Subscription streamSubscription(
```

Comment:
Can name this more simply:
```suggestion
Subscription subscription(
```

because I don't think we'll need an alternate way of making an example subscription.